packages/core/src/extensions/tiptap-extensions/Link/link.test.ts

```
@@ -0,0 +1,855 @@
+    ]);
+
+    // Place cursor without selection
+    editor.setTextCursorPosition("test-block", "end");
+    const view = editor._tiptapEditor.view;
+
+    const pastePlugin = view.state.plugins.find(
+      (p) => (p as any).key === "handlePasteLink$"
+    );
+
+    if (pastePlugin && pastePlugin.props.handlePaste) {
+      const { Slice, Fragment } = require("@tiptap/pm/model");
```

ESLint

[error] Require statement not part of import statement.

(`@typescript-eslint/no-var-requires`)

---

[error] Avoid calling `expect` conditionally`

(jest/no-conditional-expect)